ui/raidboss/data/07-dt/raid/r7s.ts

```diff
@@ -1,14 +1,421 @@
+      },
+      outputStrings: {
+        in: {
+          en: 'In at tethered wall',
+        },
+        out: {
+          en: 'Out from tethered wall',
+        },
+        inFollowup: {
+          en: 'In at tethered wall + ${followup}',
+        },
+        outFollowup: {
+          en: 'Out from tethered wall + ${followup}',
+        },
+        awayFromFront: {
+          en: 'Spread, Away from front',
+        },
+        bigAoe: Outputs.bigAoe,
+      },
+    },
+    {
+      id: 'R7S Glower Power',
+      type: 'StartsUsing',
+      netRegex: { id: 'A94C', source: 'Brute Abombinator', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Spread, Away from front',
+        },
+      },
+    },
+    {
+      id: 'R7S Revenge of the Vines',
+      type: 'StartsUsing',
+      netRegex: { id: 'A587', source: 'Brute Abombinator', capture: false },
+      response: Responses.bigAoe(),
+    },
+    {
+      id: 'R7S Thorny Deathmatch',
+      type: 'Tether',
+      netRegex: { id: '0152', capture: true },
+      infoText: (_data, matches, output) => output.text!({ target: matches.target }),
+      outputStrings: {
+        text: {
+          en: 'Tank tether on ${target}',
+        },
+      },
+    },
+    {
+      id: 'R7S Abominable Blink',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.flareMarker, capture: true },
+      alertText: (data, matches, output) => {
+        if (matches.target === data.me)
+          return output.flare!();
+        return output.avoidFlare!();
+      },
+      outputStrings: {
+        avoidFlare: {
+          en: 'Away from Flare',
+        },
+        flare: {
+          en: 'Flare on YOU, Away from party',
+        },
+      },
+    },
+    {
+      // different strats have different players taking these tethers,
+      // so we use a generic callout for now
+      id: 'R7S Demolition Deathmatch',
+      type: 'StartsUsing',
+      netRegex: { id: 'A596', source: 'Brute Abombinator', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get tethers',
+        },
+      },
+    },
+    {
+      id: 'R7S Strange Seeds',
+      type: 'StartsUsing',
+      netRegex: { id: 'A598', source: 'Brute Abombinator', capture: true },
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Drop seed',
+        },
+      },
+    },
+    {
+      id: 'R7S Tendrils of Terror',
+      type: 'StartsUsing',
+      netRegex: {
+        id: ['A599', 'A59A', 'A59C', 'A59D'],
+        source: 'Brute Abombinator',
+        capture: false,
+      },
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Avoid line AoEs',
+        },
+      },
+    },
```

Comment:
These two combined might be a bit spammy?

Reply:
During Demolition Deathmatch, some strats have some players needing to move between every seed drop, while others move only when they have a seed.  There's not a good way to handle the large amount of strat variations here.